src/MSBuild/XMake.cs

```diff
@@ -604,7 +603,12 @@ string commandLine
                         ref inputResultsCaches,
                         ref outputResultsCache,
                         ref lowPriority,
-                        recursing: false
+                        recursing: false,
+#if FEATURE_GET_COMMANDLINE
+                        commandLine
+#else
+                        string.Join(" ", commandLine)
```

Comment:
There isn't a string.Join overload for char.

Reply:
> There isn't a string.Join overload for char.

. . . until .NET Core 2.0.

Reply:
This code is under an ifdef and does not run on Framework. Just a nit and I wrote it only because you're adding a call to the `char`-taking overload in this PR (line ~1653).

Reply:
I thought I'd tested it at that location, but I was wrong. Thanks!